src/Hintman/Installation.hs

```diff
@@ -0,0 +1,86 @@
+{- | Functions for acquiring information about GitHub application installation.
+-}
+
+module Hintman.Installation
+       ( createAccessToken
+       ) where
+
+import Prelude hiding (exp)
+
+import Data.Aeson (decode)
+import Data.Time (NominalDiffTime, getCurrentTime)
+import Data.Time.Clock.POSIX (utcTimeToPOSIXSeconds)
+import Network.HTTP.Client (Request (..), Response (..), httpLbs, parseUrlThrow)
+import Network.HTTP.Client.TLS (newTlsManager)
+import Network.HTTP.Types (Status (..))
+import Web.JWT (JWTClaimsSet (..), Signer (RSAPrivateKey), encodeSigned, numericDate, stringOrURI)
+
+import Hintman.Core.Token (AppInfo (..), InstallationAccessToken, InstallationId (..))
+
+
+newtype JwtToken = JwtToken Text
```

Comment:
Shouldn't we move this to `Hintman.Core.Jwt`?

Reply:
This data type is internal and exists only to improve readability. The problem is that `encodeSigned` function from `jwt` library returns `Text`:

* http://hackage.haskell.org/package/jwt-0.10.0/docs/Web-JWT.html#v:encodeSigned

I feel uncomfortable using plain `Text` for `JwtToken`. This data type is just an internal implementation detail and it's used by the exposed interface, so I didn't bother to put it in a separate module...